src/main/drivers/accgyro_mpu3050.c

```diff
@@ -23,26 +23,20 @@
 #include "common/maths.h"
 
 #include "system.h"
+#include "exti.h"
 #include "bus_i2c.h"
 
 #include "sensor.h"
 #include "accgyro.h"
+#include "accgyro_mpu.h"
 #include "accgyro_mpu3050.h"
 
-
+extern mpuDetectionResult_t mpuDetectionResult;
```

Comment:
This is terrible idea. Use some header for extern declaration (`accgyro_mpu_impl.h`) and include that header in file where variable (and function) is defined.
When declared this way there is no check that extern type matches declaration. It is possibly source of very hard to find errors ...
(I spent few afternoons hunting similar error in my cleanflight changes some time ago)